### PR TITLE
Added asset chain testnet

### PR DESCRIPTION
**Pull request by CeoFred**
## PR-Codex overview
The focus of this PR is to add the `AssetChain Testnet` to the project's supported chains.

### Detailed summary
- Added `AssetChain Testnet` with RPC URLs and block explorers
- Defined `assetChainTestnet` in `chains/definitions/assetChainTestnet.ts`
- Updated `viem` package to version 2.19.4 with peer dependency on `typescript@5.0.4`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by KodeSage** [comment]
@jxom everything is updated now. please review.

File: pnpm-lock.yaml
Comment (jxom) [inline pnpm-lock.yaml:1]: Again, this change shouldn't be here. I'll fix it up...

**Comment by CeoFred** [comment]
Thank you @jxom

**Comment by CeoFred** [comment]
@jxom this pr was merged to main, but I cannot see the additions in there.. could you help out with this with more information. thank you

**Pull request merged**